### PR TITLE
Note MPI fork depletion analysis issues in MPI installation guide

### PR DESCRIPTION
**Pull request by twhitehead**
Spent a large amount of time discovering the MPI fork issue for one of Canadian HPC researchers. Here is a pull request to mention it in the installation section in the hopes it may help future HPC support staff (we tend not to dig into the manual much beyond building and basic running). Thanks!

### CODE REVIEW

File: docs/source/usersguide/install.rst [hunk 1]
```diff
@@ -378,6 +378,17 @@ i.e.
 
     CXX=mpicxx cmake /path/to/openmc
 
+If you are doing depletion analysis, you may have to set
+
+.. code-block:: python
+
+    openmc.pool.USE_MULTIPROCESSING = False
```
Comment (drewejohnson) [inline docs/source/usersguide/install.rst:385]: The object is `openmc.deplete.pool.USE_MULTIPROCESSING` - https://docs.openmc.org/en/stable/pythonapi/deplete.html#openmc.deplete.pool.USE_MULTIPROCESSING

And while I agree that we should make this more user facing, I don't think this section is right. Perhaps `docs/source/usersguide/troubleshoot.rst` or `docs/source/usersguide/depletion.rst`?

**Review by drewejohnson — CHANGES_REQUESTED** [review]

**Comment by twhitehead** [comment]
On the subject of where in the document it goes, my only input is to encourage putting it somewhere that someone that is only going build the software and not run it will read it.

For your typical HPC support person, installing OpenMC will just be one of many installation requests, and we wont be digging through the documentation much beyond the how to install section. Certainly not any of the detailed how to use directions.

Just my 2¢, but we really do appreciated if anything that is going to trip up having it working on a cluster can be at least be referenced up front so we are made aware of it.

Thank!  -Tyson

**Comment by paulromano** [comment]
@twhitehead First off thanks for the contribution here -- much appreciated! I also appreciate your points on this being visible to software maintainers. However, this flag is really a runtime thing (not something that you choose when installing). As you said, as long as there's a way to reference this in the documentation it should be sufficient. I think this probably makes the most sense to be in our troubleshooting section (`docs/source/usersguide/troubleshoot.rst`) as @drewejohnson suggested.

**Comment by twhitehead** [comment]
Not a problem. I'll move it into the troubleshooting section and do another push. :+1:

**Comment by paulromano** [comment]
@twhitehead I just put in a new PR that builds off this one (rebased to our `develop` branch and moved what you had written to a different section in the documentation) so I'll go ahead and close this.

**Comment by twhitehead** [comment]
Thanks. And sorry this accidentally slipped of my radar and I never followed up with another push as I said I was going to.

**Comment by paulromano** [comment]
No worries -- thanks again for contributing to the documentation!

**Pull request closed without merging**